cardano-cli/src/Cardano/CLI/Byron/Genesis.hs

```diff
@@ -15,12 +15,8 @@ where
 import           Cardano.Api (Key (..), NetworkId, writeSecrets)
```

Comment:
Not for this PR but the fact we are exposing `Cardano.Api` here could mean byron related things are being exported via `Cardano.Api`